build.sbt

```diff
@@ -106,19 +106,18 @@ val sharedSettings = Seq(
   publishMavenStyle := true,
   Test / publishArtifact := false,
   pomIncludeRepository := { x => false },
-  releaseProcess := Seq[ReleaseStep](
-    checkSnapshotDependencies,
-    inquireVersions,
-    runClean,
-    runTest,
-    setReleaseVersion,
-    commitReleaseVersion,
-    tagRelease,
-    publishArtifacts,
-    setNextVersion,
-    commitNextVersion,
-    ReleaseStep(action = Command.process("sonatypeReleaseAll", _)),
-    pushChanges
+  releaseProcess := (
+    if (version.value.trim.endsWith("SNAPSHOT"))
+      Seq[ReleaseStep](
+        inquireVersions,
+        runClean,
+        publishArtifacts,
+        ReleaseStep(action = Command.process("sonatypeReleaseAll", _)),
+      )
+    // todo : implement workflow for final tagged releases that are not snapshots
+    else Seq[ReleaseStep](
+
+    )
```

Comment:
this will be in follow-up PR once we have snapshots working well off develop

Reply:
I don't think we need to run tests as we already run tests on each PR so that should be fine